test/root/predicates/ERC20Predicate.test.js

```diff
@@ -116,7 +146,7 @@ contract('ERC20Predicate', async function(accounts) {
       const startExitTx = await utils.startExitForErc20PredicateLegacy(contracts.ERC20Predicate.startExitForOutgoingErc20Transfer, headerNumber, blockProof, block.number, block.timestamp, reference, 1, /* logIndex */ exitTx)
       const logs = logDecoder.decodeLogs(startExitTx.receipt.rawLogs)
       // console.log(startExitTx, logs)
-      const log = logs[1]
+      const log = logs[utils.filterEvent(logs, 'ExitStarted')]
```

Comment:
this should be hardcoded, we should expect things at the exact location. since it's a test it doesn't matter regarding those hardcoded values.